lib/index.coffee

```diff
@@ -7,19 +7,24 @@ _ = require 'lodash'
 { createCompare, lruSort } = require './lru'
 dockerUtils = require './docker'
 
-current_mtimes = {}
+class DockerGC
+	setDocker: (hostObj) ->
+		@currentMtimes = {}
+		@hostObj = hostObj
+		dockerUtils.getDocker(hostObj)
+		.then (docker) =>
```

Comment:
Did you know, this can be
```coffee
		.then (@docker) =>
```
(I think this one is potentially a tad obscure though)